README.md

```diff
@@ -4,10 +4,82 @@ This project aims to provide the [containerd shim](https://github.com/containerd
 
 [runwasi](https://github.com/deislabs/runwasi) is a project that aims to run WASI workloads managed by [containerd](https://containerd.io/).
 
+## Shim and Spin Version Map
+
+Below is a table for referencing the version of the Spin runtime used in each `containerd-shim-spin` release.
+
+| **shim version** | v0.12.0                                                                          | v0.13.0                                                                            | v0.14.0                                                                             | v0.14.1                                                                             |
+| ---------------- | -------------------------------------------------------------------------------- | -------------------------------------------------------------------------------- | -------------------------------------------------------------------------------- | -------------------------------------------------------------------------------- | 
+| **spin**         | [v2.2.0](https://github.com/fermyon/spin/releases/tag/v2.2.0)                    | [v2.3.1](https://github.com/fermyon/spin/releases/tag/v2.3.1)                    | [v2.4.2](https://github.com/fermyon/spin/releases/tag/v2.4.2)                    | [v2.4.3](https://github.com/fermyon/spin/releases/tag/v2.4.3)                    |
+
 ## Documentation
 
 To learn more about the Containerd Shim Spin, please visit [the official Containerd Shim Spin documentation](https://www.spinkube.dev/docs/containerd-shim-spin/).
 
+## Installing the `containerd-shim-spin` on Kubernetes Nodes
+
+In order to run Spin applications on your cluster, you must complete the following three steps:
+
+1. Install the shim on each Node that should support Spin apps
+2. Update the containerd configuration to recognize the shim
+3. Apply the Kubernetes `RuntimeClass` for the shim
+
+Repeating steps 1 and 2 for each node on a cluster can be a timely and manual process. For this reason, SpinKube provides a [`runtime-class-manager`](https://www.spinkube.dev/docs/runtime-class-manager/) (previously the `kwasm` operator) that enables you to skip over step 1 and 2. See the [SpinKube installation guide](https://www.spinkube.dev/docs/spin-operator/installation/installing-with-helm/) for more information on installing with Helm.
+
+To carry out the installation step-by-step, do the following:
+
+1. Install the shim on each Node that should support Spin apps
+
+    Install a [release of the shim](https://github.com/spinkube/containerd-shim-spin/releases) somewhere on the path of your Kubernetes worker nodes. For example, copy `containerd-shim-spin-v2` to `/bin`. Shims are additive, so once the `containerd-shim-spin` is installed on a Node, it can support Spin WebAssembly apps alongside traditional containerized deployments. 
```

Comment:
```suggestion
    Install a [release of the shim](https://github.com/spinkube/containerd-shim-spin/releases) on the `PATH` of your Kubernetes worker nodes. For example, copy `containerd-shim-spin-v2` to `/bin`. Shims are additive, so once the `containerd-shim-spin` is installed on a Node, it can support Spin WebAssembly apps alongside Linux containers. 
```